_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -69,121 +71,140 @@ In order to use the `s3` sink, configure AWS Identity and Access Management (IAM
 
-Because Avro requires a schema, you may either define the schema yourself, or Data Prepper will automatically generate a schema.
-In general, you should define your own schema because it will most accurately reflect your needs.
+When you provide your own Avro schema, that schema defines the final structure of your data. Any extra values inside any incoming events that are not mapped in the Arvo schema will not be included in the final destination. Data Prepper does not allow the use of the `include_keys` or `exclude_keys` with a custom schema. This is to avoid confusion between a custom Arvo schema and the `include_keys` or `exclude_keys` sink configurations. 
 
-We recommend that you make your Avro fields use a null [union](https://avro.apache.org/docs/current/specification/#unions).
-Without the null union, each field must be present or the data will fail to write to the sink.
-If you can be certain that each each event has a given field, you can make it non-nullable.
+In cases where your data is uniform, you may be able to automatically generate a schema. Auto-generated schemas are based on the first event that the codec receives. The schema will only contain keys from this event, and you must have all keys present in all events in order for the auto-generated schema to produce a working schema. Auto-generated schemas make all fields nullable. Use the `include_keys` and `exclude_keys` sink configurations to control what data is included in the auto-generated schema.
 
-When you provide your own Avro schema, that schema defines the final structure of your data.
-Therefore, any extra values inside any incoming events that are not mapped in the Arvo schema will not be included in the final destination.
-To avoid confusion between a custom Arvo schema and the `include_keys` or `exclude_keys` sink configurations, Data Prepper does not allow the use of the `include_keys` or `exclude_keys` with a custom schema.
-
-In cases where your data is uniform, you may be able to automatically generate a schema.
-Automatically generated schemas are based on the first event received by the codec.
-The schema will only contain keys from this event.
-Therefore, you must have all keys present in all events in order for the automatically generated schema to produce a working schema.
-Automatically generated schemas make all fields nullable.
-Use the sink's `include_keys` and `exclude_keys` configurations to control what data is included in the auto-generated schema.
+Avro fields should use a null [union](https://avro.apache.org/docs/current/specification/#unions), as this will allow missing values. Otherwise, all required fields must be present for each event. Use non-nullable fields only when you are certain they exist.
```

Comment:
```suggestion
Avro fields should use a null [union](https://avro.apache.org/docs/current/specification/#unions) because this will allow missing values. Otherwise, all required fields must be present for each event. Use non-nullable fields only when you are certain they exist.
```